third_party/binutils/test_decenc/test_decenc_x86.asm

```diff
@@ -139588,6 +139588,18 @@ GLOBAL_LABEL(FUNCNAME:)
         RAW(7b) RAW(00) RAW(00) RAW(00)
         RAW(62) RAW(f2) RAW(7d) RAW(29) RAW(91) RAW(b4) RAW(f5)
         RAW(7b) RAW(00) RAW(00) RAW(00)
+        END_OF_SUBTEST_MARKER
+
+        /* xsaves.s */
+        RAW(0f) RAW(c7) RAW(28)
+        RAW(0f) RAW(c7) RAW(ac) RAW(f4) RAW(c0) RAW(1d) RAW(fe)
+        RAW(ff)
+        END_OF_SUBTEST_MARKER
+
+        /* xrstors.s */
```

Comment:
Are you sure there's not an existing disabled test, like there is for xsaves?

Reply:
Didn't find "xrstors" string or similar bytes that are disabled.